### PR TITLE
Release v0.0.3

### PR DESCRIPTION
**Pull request by christophfink**
Changed version number and added CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+- **0.0.3** (2022-05-12):
+    - refreshed documention
+    - set up GitHub extas
```
[the remark this answers is not in the document]
Comment (HTenkanen) [reply CHANGELOG.md:3]: Also possible additions:
- added contribution guidelines
- added issue templates
- added tests
- setup codecov.io

**Review by HTenkanen — APPROVED** [review]
Super!

**Comment by HTenkanen** [comment]
@christophfink After the tests, is this now ready to go?

**Comment by christophfink** [comment]
yeah, I’d think so

**Comment by HTenkanen** [comment]
Wohoo! 🎉

**Pull request merged**